esy.json

```diff
@@ -0,0 +1,23 @@
+{
+  "dependencies": {
+    "bs-platform": "*"
+  },
+  "resolutions": {
+    "bs-platform": "EduardoRFS/bucklescript#1884423622d4bd21b34f2df472c245aae924206d",
+    "ocaml": "EduardoRFS/ocaml#73143a0348dfec8c5fb382ffbc10e0cb242fbdb3"
+  },
+  "esy": {
+    "buildsInSource": "_build",
+    "build": [
+      "ln -sfn #{bs-platform.install} node_modules/bs-platform",
+      "sh -c \"echo '(lang dune 2.7)' > node_modules/bs-decoders/dune-project\"",
```

Comment:
That's a hack, because bs-decoders ships with `(lang dune 1.0)` which is not compatible, with the current dune generated from bsb